modules/storage-persistent-storage-pv.adoc

```diff
@@ -49,6 +49,7 @@ ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 - Fibre Channel
 endif::openshift-enterprise,openshift-webscale,openshift-origin[]
 - GCE Persistent Disk
+- GCP Filestore
```

Comment:
We should probably be consistent with the naming in lines 51 and 52. I'd vote for changing line 51 (rename `GCE` to `GCP`).

Reply:
Done.